src/main/features/response/form/models/statement-of-means/numberOfPeople.ts

```diff
@@ -0,0 +1,47 @@
+import { IsDefined, IsInt, Min } from 'class-validator'
+import { toNumberOrUndefined } from 'common/utils/numericUtils'
+import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
+import { MaxLength } from 'forms/validation/validators/maxLengthValidator'
+import { IsNotBlank } from 'forms/validation/validators/isBlank'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+
+export class ValidationErrors {
+  static readonly NUMBER_OF_PEOPLE_REQUIRED: string = 'Enter a number of people you support'
+  static readonly DETAILS_REQUIRED: string = 'Enter details'
+}
+export class NumberOfPeople {
+
+  @IsDefined({ message: ValidationErrors.NUMBER_OF_PEOPLE_REQUIRED })
+  @IsInt({ message: GlobalValidationErrors.INTEGER_REQUIRED })
+  @Min(1, { message: GlobalValidationErrors.POSITIVE_NUMBER_REQUIRED })
+  value: number
+
+  @IsDefined({ message: ValidationErrors.DETAILS_REQUIRED })
+  @IsNotBlank({ message: ValidationErrors.DETAILS_REQUIRED })
+  @MaxLength(ValidationConstraints.FREE_TEXT_MAX_LENGTH, { message: GlobalValidationErrors.TOO_LONG_INPUT })
```

Comment:
`TOO_LONG_INPUT` is a duplicate -- see `ValidationErrors#FREE_TEXT_TOO_LONG`. It may be not you duplicating it but please fix that.

Reply:
FREE_TEXT_TOO_LONG renamed to TEXT_TOO_LONG
all usages of TOO_LONG_INPUT replaced with TEXT_TOO_LONG